modules/markup/sanitizer.go

```diff
@@ -22,7 +22,10 @@ type Sanitizer struct {
 	init             sync.Once
 }
 
-var sanitizer = &Sanitizer{}
+var (
+	sanitizer     = &Sanitizer{}
+	allowAllRegex = regexp.MustCompile(".+")
```

Comment:
Would `".+:.+"` work as well? Just a tiny bit stricter.

Reply:
It doesn't work, the scheme itself doesn't contain `:`

Reply:
Ah, so it's splitting at the first `:`, I suppose.

Reply:
Interestingly, this parsing is different than how the WHATWG URL standard does it, where `protocol` does include the `:`, e.g. `https:`.